src/main/java/step3/racingCar/GameFactory.java

```diff
@@ -15,19 +15,33 @@
 
 public class GameFactory {
 
-    public static Game game(){
-        return new Game();
+    public static final String EMPTY_NAME = "emptyName";
+
+    public static Game game(String[] str) {
+        return new Game(str);
+    }
+
+    public static Cars namesCars(String[] names) {
+        List<Car> carList = new ArrayList<>();
+        for (String name : names) {
+            carList.add(car(name));
+        }
+        return new Cars(carList);
```

Comment:
```suggestion
        List<Car> carList = Arrays.stream(names).map(GameFactory::car).collect(Collectors.toList());
        return new Cars(carList);
```
이렇게도 표현 가능한 것 같아요 :)